outputs.tf

```diff
@@ -13,6 +13,11 @@ output "storage_bucket_arn" {
   description = "Bucket ARN"
 }
 
+output "iam_role" {
+  description = "IAM Role"
```

Comment:
maybe add more specific description what is the role for.
"IAM Role" is too generic

Reply:
b9f44f1291121dfe95ded8bf1db52a70084fb773